### DISCUSSION TITLE
Can dp model predict the extremely small number of atoms in system?

### DISCUSSION THREAD

**Discussion by lhycms** [deepmodeling/deepmd-kit#3778] [Q&A]
If the number of element A in a system is 180, the number of element B is 300, and the number of element C is 18, can the dp model accurately predict the energy and force of element C? Does deepmd-kit process this situation specially?
I have tried MTP, it can't accurately predict the energy and force of element C. 
Thanks.

**Comment by njzjz** [comment]
Could you explain what the energy of element C is? I don't think the energy of a single atom in a system has any physical meaning.

**Comment by chtchelkatchev** [comment]
The development of Machine Learning Interaction Potential (MLIP) for the study of alloyed alloys with low concentrations of alloying components is a tricky task. The traditional MLIP machine learning procedure does not converge well in this case, since the neural network contains “untrainable” parameters that characterize the interaction of all types of atoms, including alloying ones, which rarely close approach each other. We propose a solution - in a certain way to add to the training database a small number of "high-entropy" configurations with a high concentration of alloying elements, where these atoms approach and interact with a fairly high probability. As a scientific backlog, we created MLIP by this method for 5-component FeBSiTaNb alloys, where the concentrations of Ta and Nb are less than 1%, and with its help we were able to explain the experimental data on structure-sensitive properties [I.V. Sterkhova et al., https://doi.org/10.48550/arXiv.2211.03500]; see also our similar work on Al86Ni2Co6Gd6 alloys [https://arxiv.org/abs/2211.03513].
![Рисунок1](https://github.com/deepmodeling/deepmd-kit/assets/56683266/629efeed-868d-4ebf-b1ab-3b5e7b6355e0)
